flight_computer/src/usb/msc/emfat_file.cpp

```diff
@@ -54,31 +54,31 @@ static void lfs_read_file(uint8_t *dest, int size, uint32_t offset, emfat_entry_
     }
 
     // Assume the files starting with 'f' are flight logs; all others are considered to be stats files.
-    const bool flight_log = entry->name != NULL && entry->name[0] == 'f';
+    const bool flight_log = entry->name != nullptr && entry->name[0] == 'f';
     snprintf(filename, 32, flight_log ? "/flights/flight_%05hu" : "/stats/stats_%05hu.txt", entry->lfs_flight_idx);
-    int err = lfs_file_open(&lfs, &curr_file, filename, LFS_O_RDONLY);
-    if (err) {
+    const int err = lfs_file_open(&lfs, &curr_file, filename, LFS_O_RDONLY);
+    if (err < 0) {
```

Comment:
!= 0?

Reply:
From lfs documentation:

```
// Returns a negative error code on failure.
int lfs_file_open(lfs_t *lfs, lfs_file_t *file, const char *path, int flags);
```

So we should check for `err < 0`.